projectatomic/docker
# Change login success message and server selection

Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

**- What I did**
Docker login now takes the registry from /etc/containers/registries.conf if there's one and only one entry for registries in that file.  Otherwise it defaults to docker.io.  In either case the "Login Succeeded" message has been append with "to {servername}" to let the user know where the login was completed to.

**- How I did it**
vi is my friend.  Followed by testing.

**- How to verify it**
`docker login` should show the registry that it logged into upon successful completion.  If there is not just one registry listed in /etc/containers/registries.conf registries.search registries[] table and a server is not provided to `docker login`, then docker.io will be use as the server target.

**- Description for the changelog**


Testing:
```
# docker login
Login with your Docker ID to push and pull images from Docker Hub. If you don't have a Docker ID, head over to https://hub.docker.com to create one.
Username (tomsweeneyredhat):         
Password: 
Login Succeeded to https://registry-1.docker.io

```


**- A picture of a cute animal (not mandatory but encouraged)**
